components/03-modules/review/review--add.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="review--add {{ class }}" {{{ attributes }}}>
+    <div class="review__divider">
+        {{ render '@rating--static' rating }}
```

Comment:
Just curious? What `static` means? is it for stars summary?

Reply:
static means no hover effect on start, I can rename it for ex. to 'summary'

Reply:
please, rename it for `summary` it's some naming connecting to magento, will be more understood

Reply:
done